packager/amp_cache_transform/amp_cache_transform_test.go

```diff
@@ -0,0 +1,23 @@
+package amp_cache_transform
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestShouldSendSXG(t *testing.T) {
+	assert.True(t, ShouldSendSXG("any"))
+	assert.True(t, ShouldSendSXG("google"))
+	assert.True(t, ShouldSendSXG("google, any"))
+	assert.True(t, ShouldSendSXG("google, foobar"))
+	assert.True(t, ShouldSendSXG("foobar, any"))
```

Comment:
"google,foobar"

Verify that space isn't significant. Also try with "google,\tfoobar" "google ,foobar"

assert.False(t, ShouldSendSXG("googleany"))

Verify that the code isn't just performing substring matching.

Verify some of the error checking for identifiers:

False: "google,123"
False: "ABC,google"

Reply:
Done.